v4/source/definitions.yaml

```diff
@@ -786,6 +786,24 @@ components:
         url:
           description: The URL that is triggered
           type: string
+    AutocompleteSuggestion:
+      type: object
+      properties:
+        Complete:
+          description: Completed suggestion 
+          type: string
+        Suggestion:
+          description: Predicted text user might want to input
+          type: string
+        Hint:
+          description: Hint about suggested input
+          type: string
+        Description: 
+          description: Description of the suggested command
+          type: string
+        IconData:
+          description: Icon data in base64 format
```

Comment:
The documentation should make clear that this is an scg image.

Reply:
see my comment in the source about maybe renaming to be more intention-revealing